Hackney.Shared.HousingSearch/Domain/Transactions/SuspenseResolutionInfo.cs

```diff
@@ -4,11 +4,19 @@ namespace Hackney.Shared.HousingSearch.Domain.Transactions
 {
     public class SuspenseResolutionInfo
     {
+        public SuspenseResolutionInfo(DateTime? resolutionDate, bool isConfirmed, bool isApproved, string note)
```

Comment:
I would make this private, and add:

 `public static SuspenseResolutionInfo Create(DateTime? resolutionDate, bool isConfirmed, bool isApproved, string note)` to return a new instance of the class

Reply:
What's the difference between these two approaches?

Reply:
Changed to use static factory method